CHANGELOG.md

```diff
@@ -29,6 +29,11 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 - The `NewMetricExporter` in `go.opentelemetry.io/otel/bridge/opencensus` is deprecated.  Use `NewMetricProducer` instead. (#3541)
 
+### Changed
```

Comment:
This duplicates the `### Changed` section. This needs to be merged there.

Reply:
I had been confused by that. It turns out it's the automatic merge that was creating the duplicate section, so I couldn't reproduce it locally.